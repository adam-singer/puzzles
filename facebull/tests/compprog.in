M1 C1 C2 5
M2 C2 C3 1
M3 C3 C5 6
M4 C1 C4 3
M5 C4 C5 2
M6 C4 C2 2
M7 C2 C4 3
M8 C4 C3 9
M9 C5 C3 4
M10 C5 C1 7
