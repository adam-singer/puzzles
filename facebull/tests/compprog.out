19
2 3 4 6 10
